db/seeds.d/170-notification_blueprints.rb

```diff
@@ -61,6 +61,26 @@
     message: N_('Support for %{feature} has been deprecated and will be removed in version %{version}'),
     expires_in: 30.days,
   },
+  {
+    group: N_('Reports'),
+    name: 'report_finish',
+    message: N_('Template is ready to download'),
```

Comment:
Is the template ready or a report?
```suggestion
    message: N_('Report is ready to download'),
```